uppsrc/ide/ide.lay

```diff
@@ -1,7 +1,7 @@
 LAYOUT(SelectPackageLayout, 816, 468)
 	ITEM(Upp::ParentCtrl, parent, HSizePosZ(4, 8).VSizePosZ(4, 32))
-	ITEM(Upp::DropList, kind, Tip(t_("Select which packages should be displayed")).LeftPosZ(4, 124).BottomPosZ(7, 19))
-	ITEM(Upp::DropList, nest, Tip(t_("Select which packages should be displayed")).LeftPosZ(132, 112).BottomPosZ(7, 19))
+	ITEM(Upp::DropList, kind, Tip(t_("Select which packages should be displayed")).LeftPosZ(4, 124).BottomPosZ(6, 20))
+	ITEM(Upp::DropList, nest, Tip(t_("Select which packages should be displayed")).LeftPosZ(132, 112).BottomPosZ(6, 20))
```

Comment:
What is the issue here? Can you explain or optimally put screenshot before/after.